README.md

```diff
@@ -2,42 +2,85 @@
 
 Consider donating to support development: https://ko-fi.com/johnbalis
 
-
 ## About
-This is a libreoffice writer extension to allow for inline generative editing with local inference. It can be used with any language model supported by text-generation-webui. 
 
-This extension adds two powerful commands to libreoffice writer:
+This is a LibreOffice Writer extension that enables inline generative editing with local inference. It's compatible with language models supported by `text-generation-webui` and `ollama`.
+
+## Table of Contents
+
+*   [About](#about)
+*   [Table of Contents](#table-of-contents)
+*   [Features](#features)
+    *   [Extend Selection](#extend-selection)
+    *   [Edit Selection](#edit-selection)
+*   [Setup](#setup)
+    *   [LibreOffice Extension Installation](#libreoffice-extension-installation)
+    *   [Backend Setup](#backend-setup)
+        *   [text-generation-webui](#text-generation-webui)
+        *   [Ollama](#ollama)
+*   [Settings](#settings)
+*   [License](#license)
+
+## Features
+
+This extension provides two powerful commands for LibreOffice Writer:
 
 ### Extend Selection
-Uses a language model to predict what comes after the selected text. There are a lot of ways to use this.
 
-Some example use cases for this include, writing a story or an email given a particular prompt, adding additional possible items to a grocery list, or summarizing the selected text. 
+**Hotkey:** `CTRL + q`
+
+This uses a language model to predict and generate text following the selected content.  This is useful for tasks like writing stories, emails, or expanding on existing text.
 
 ### Edit Selection
-A dialog box appears to prompt the user for instructions about how to edit the selected text, then the selected text is replaced by the edited text. 
 
-Some examples for use cases for this include changing the tone of an email, translating text to a different language, and semantically editing a scene in a story. 
+**Hotkey:** `CTRL + e`
+
+Allows users to prompt a language model to edit the selected text. The model will generate edited text, which replaces the original selection.  This is useful for tasks like changing the tone of an email or performing semantic editing.
```

Comment:
Please revert the text to this:


* A dialog box appears to prompt the user for instructions about how to edit the selected text, then the selected text is replaced by the edited text.

* Some examples for use cases for this include changing the tone of an email, translating text to a different language, and semantically editing a scene in a story.